assets/dev/js/frontend/utils/lightbox/lightbox.js

```diff
@@ -264,32 +276,50 @@ module.exports = elementorModules.ViewModule.extend( {
 
 		elements.$header = $( '<header>', { class: slideshowClasses.header + ' ' + classes.preventClose } );
 
-		if ( showCounter ) {
-			elements.$counter = $( '<span>', { class: slideshowClasses.counter } );
-			elements.$header.append( elements.$counter );
-		}
-
-		if ( showFullscreen ) {
-			elements.$iconExpand = $( '<i>', { class: slideshowClasses.iconExpand } ).append( $( '<span>' ), $( '<span>' ) );
-			elements.$iconExpand.on( 'click', this.toggleFullscreen );
-			elements.$header.append( elements.$iconExpand );
-		}
-
-		if ( showZoom ) {
-			elements.$iconZoom = $( '<i>', { class: slideshowClasses.iconZoomIn } );
-			elements.$iconZoom.on( 'click', this.toggleZoomMode );
-			elements.$header.append( elements.$iconZoom );
-		}
-
 		if ( showShare ) {
-			elements.$iconShare = $( '<i>', { class: slideshowClasses.iconShare } ).append( $( '<span>' ) );
+			elements.$iconShare = $( '<i>', {
+				class: slideshowClasses.iconShare,
+				role: 'button',
+				'aria-label': i18n.share,
+				'aria-expanded': false,
+			} ).append( $( '<span>' ) );
 			const $shareLinks = $( '<div>' );
 			$shareLinks.on( 'click', ( e ) => {
 				e.stopPropagation();
 			} );
 			elements.$shareMenu = $( '<div>', { class: slideshowClasses.shareMenu } ).append( $shareLinks );
 			elements.$iconShare.add( elements.$shareMenu ).on( 'click', this.toggleShareMenu );
 			elements.$header.append( elements.$iconShare, elements.$shareMenu );
+			this.buttons.push( elements.$iconShare[ 0 ] );
+		}
+
+		if ( showZoom ) {
+			elements.$iconZoom = $( '<i>', {
+				class: slideshowClasses.iconZoomIn,
+				role: 'switch',
+				'aria-checked': false,
+				'aria-label': i18n.zoom,
+			} );
+			elements.$iconZoom.on( 'click', this.toggleZoomMode );
+			elements.$header.append( elements.$iconZoom );
+			this.buttons.push( elements.$iconZoom[ 0 ] );
+		}
+
+		if ( showFullscreen ) {
+			elements.$iconExpand = $( '<i>', {
+				class: slideshowClasses.iconExpand,
+				role: 'switch',
+				'aria-checked': false,
+				'aria-label': i18n.fullscreen,
+			} ).append( $( '<span>' ), $( '<span>' ) );
+			elements.$iconExpand.on( 'click', this.toggleFullscreen );
+			elements.$header.append( elements.$iconExpand );
+			this.buttons.push( elements.$iconExpand[ 0 ] );
+		}
```

Comment:
It would be better to create a method that builds icon, attaches its events and pushes it to the buttons array

Reply:
Myabe another time